### PR TITLE
Update Numeric Entry and UpDown UG Documentation

### PR DESCRIPTION
**Pull request by surya3655**

### CODE REVIEW

File: maui-toolkit-toc.html [hunk 1]
```diff
@@ -245,6 +245,7 @@
             <li><a href="/maui-toolkit/NumericEntry/Basic-Features">Basic Features</a></li>
             <li><a href="/maui-toolkit/NumericEntry/Formatting">Formatting</a></li>
             <li><a href="/maui-toolkit/NumericEntry/Restriction">Restriction</a></li>
+            <li><a href="/maui-toolkit/NumericEntry/Events">Restriction</a></li>
```
Comment (Choza-rajan) [inline maui-toolkit-toc.html:248]: Use the topic name also Events instead of Restriction

File: maui-toolkit-toc.html [hunk 2]
```diff
@@ -256,6 +257,7 @@
             <li><a href="/maui-toolkit/NumericUpDown/Formatting">Formatting</a></li>
             <li><a href="/maui-toolkit/NumericUpDown/Restriction">Restriction</a></li>
             <li><a href="/maui-toolkit/NumericUpDown/Updown-button">UpDown Button</a></li>
+            <li><a href="/maui-toolkit/NumericUpDown/Events">UpDown Button</a></li>
```
Comment (Choza-rajan) [inline maui-toolkit-toc.html:260]: Use the topic name also Events instead of UpDown Button

File: maui-toolkit-toc.html (hunk 1 above)
Comment (surya3655) [reply maui-toolkit-toc.html]: updated

File: maui-toolkit-toc.html (hunk 2 above)
Comment (surya3655) [reply maui-toolkit-toc.html]: updated

File: maui-toolkit/NumericUpDown/Events.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+{% endhighlight %}
+{% endtabs %}
+
+You can handle the event as follows.
+
+{% tabs %}
+{% highlight C# %}
+
+private void sfNumericUpDown_ValueChanged(object sender, NumericUpDownValueChangedEventArgs e)
```
Comment (Choza-rajan) [inline maui-toolkit/NumericUpDown/Events.md:44]: recheck the arguments

File: maui-toolkit/NumericEntry/Events.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                        VerticalOptions="Center"
+                        Completed="sfNumericEntry_Completed" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumericEntry sfNumericEntry = new SfNumericEntry();
+sfNumericEntry.HorizontalOptions = LayoutOptions.Center;
+sfNumericEntry.VerticalOptions = LayoutOptions.Center;
+sfNumericEntry.ValueChanged += sfNumericEntry_Completed;
```
[the remark this answers is not in the document]
Comment (surya3655) [reply maui-toolkit/NumericEntry/Events.md]: I have corrected it

File: maui-toolkit/NumericUpDown/Events.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+                        VerticalOptions="Center"
+                        Completed="sfNumericUpDown_Completed" />
+
+{% endhighlight %}
+{% highlight C# %}
+
+SfNumericUpDown sfNumericUpDown = new SfNumericUpDown();
+sfNumericUpDown.HorizontalOptions = LayoutOptions.Center;
+sfNumericUpDown.VerticalOptions = LayoutOptions.Center;
+sfNumericUpDown.ValueChanged += sfNumericUpDown_Completed;
```
[the remark this answers is not in the document]
Comment (surya3655) [reply maui-toolkit/NumericUpDown/Events.md]: I have corrected it

File: maui-toolkit/NumericUpDown/Events.md (hunk 3 above)
Comment (surya3655) [reply maui-toolkit/NumericUpDown/Events.md]: I have corrected it

File: maui-toolkit/NumericEntry/Getting-Started.md [hunk 6]
```diff
@@ -9,18 +9,14 @@ documentation: ug
 
 # Getting Started with .NET MAUI Numeric Entry
 
-This section guides you through setting up and configuring a [Numeric Entry]() in your .NET MAUI application. Follow the steps below to add a basic Numeric Entry to your project.
-
-To quickly get started with the .NET MAUI Numeric Entry, watch this video.
-
-{% youtube "https://www.youtube.com/watch?v=BHw8UpoUmoM&ab_channel=Syncfusion%2CInc" %}
+This section guides you through setting up and configuring a [Numeric Entyr]() in your .NET MAUI application. Follow the steps below to add a basic Numeric Entry Down to your project.
```
Comment (Choza-rajan) [inline maui-toolkit/NumericEntry/Getting-Started.md:12]: Numeric Entyr -- spell error

Comment (surya3655) [reply maui-toolkit/NumericEntry/Getting-Started.md]: corrected

**Review by Choza-rajan — APPROVED** [review]

**Pull request merged**